src/store/reader.rs

```diff
@@ -107,3 +123,43 @@ fn split_file(data: FileSlice) -> io::Result<(FileSlice, FileSlice, DocId)> {
         max_doc,
     ))
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::{directory::RAMDirectory, store::tests::write_lorem_ipsum_store, Directory};
+    use std::path::Path;
+
+    #[test]
+    fn test_store_lru_cache() -> crate::Result<()> {
+        let directory = RAMDirectory::create();
+        let path = Path::new("store");
+        let store_wrt = directory.open_write(path)?;
```

Comment:
I think you can write into a Vec<u8>, and wrap it into an OwnedBytes...
I believe OwnedBytes impl FileHandle.